src/codegen/math/float/tanh.rs

```diff
@@ -0,0 +1,112 @@
+//! Vertical floating-point `tanh`
+#![allow(unused)]
+
+// FIXME 64-bit 1 elem vectors tanh
+
+use libm::{
+    F32Ext,
+    F64Ext
+};
+use crate::*;
+
+crate trait Tanh {
+    fn tanh(self) -> Self;
+}
+
+macro_rules! define_tanh {
+    ($name:ident, $type:ty, $lanes:expr) => {
+        fn $name(x: $type) -> $type {
+            let mut buf = [0.0; $lanes];
+            x.write_to_slice_aligned(&mut buf);
```

Comment:
`buf` is not guaranteed to be properly aligned AFAICT - did you copy this line from somewhere ?

Reply:
No, I didn't. I just wasn't thinking about the alignment of `buf` at all. Will change that.